modules/n1ql/partials/nav.adoc

```diff
@@ -21,7 +16,7 @@
   *** xref:n1ql:n1ql-language-reference/groupby-aggregate-performance.adoc[]
   *** xref:learn:services-and-indexes/indexes/early-filters-and-pagination.adoc[]
   *** xref:learn:services-and-indexes/indexes/index-replication.adoc[]
-  *** xref:learn:services-and-indexes/indexes/storage-modes.adoc[]
+//  *** xref:learn:services-and-indexes/indexes/storage-modes.adoc[]
```

Comment:
Not sure if this should be embargoed or just removed.

Reply:
Updated to include this page and cut down the content as outlined above.

@simon-dew can you confirm its accuracy?